packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -67,30 +74,76 @@ const StyledTooltipButton = styled(TooltipButton)`
 `;
 
 interface CoinjoinSetupStrategiesProps {
-    account: Account;
+    selectedAccount: SelectedAccountLoaded;
 }
 
-export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProps) => {
+export const CoinjoinSetupStrategies = ({ selectedAccount }: CoinjoinSetupStrategiesProps) => {
     const [strategy, setStrategy] = useState<CoinJoinStrategy>('recommended');
+    const [customMaxFee, setCustomMaxFee] = useState(DEFAULT_MAX_MINING_FEE);
+    const [customSkipRounds, setCustomSkipRounds] = useState(true);
     const [connectedConfirmed, setConnectedConfirmed] = useState(false);
     const [termsConfirmed, setTermsConfirmed] = useState(false);
 
     const actions = useActions({
         startCoinjoinSession,
     });
+    const coordinatorData = useSelector(
+        state => state.wallet.coinjoin.clients[selectedAccount.network.symbol],
+    );
+    const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
+    const { notAnonymized } = useSelector(selectCurrentCoinjoinBalanceBreakdown);
```

Comment:
just note for followup. lets create a selector in `@wallet-reducers/coinjoinReducer` which returns all data combined:
- selectedAccount
- coinjoin account related to selected account
- coinjoin session of coijoin account
- coinjoin client